upup/pkg/fi/cloudup/vsphere/vsphere_cloud.go

```diff
@@ -36,6 +40,21 @@ func (c *VSphereCloud) ProviderID() fi.CloudProviderID {
 	return fi.CloudProviderVSphere
 }
 
+func NewVSphereCloud(spec *kops.ClusterSpec) (*VSphereCloud, error) {
+	server := spec.CloudConfig.VSphereServer
```

Comment:
I would suggest to convert all these flags to env variables. If not, atleast keep the VC IP as env variable.